src/main/java/org/sonarlint/intellij/actions/SonarLintToolWindow.java

```diff
@@ -82,6 +83,7 @@ public SonarLintToolWindow(Project project) {
    * Must run in EDT
    */
   public void openReportTab(AnalysisResult analysisResult) {
+    ApplicationManager.getApplication().assertIsDispatchThread();
```

Comment:
We are asserting edt here but sonarlintcheckinhandler does no utilize UI thread with showChangedFilesTab